include/tvm/relay/transform.h

```diff
@@ -165,6 +255,9 @@ class PassNode : public RelayNode {
    */
   virtual Module operator()(const Module& mod) const = 0;
```

Comment:
no need to use virtual here, as it calls into the general version using the thread local pass context

Reply:
But we need late binding, right? I think without `virtual` we will call into this function if we have `Pass = new ModulePass(xx)`.

Reply:
The implementation of ```operator()(mod)``` is the same across all passes. We just implement it as 
```
operator(mod, PassContext::Current());
```

Reply:
I see what you are saying. Thanks.